server/routes/index.ts

```diff
@@ -0,0 +1,7 @@
+import express from 'express';
+import APIV1Router from './api-v1';
+const router = express.Router();
+
+router.use('/v1', APIV1Router)
```

Comment:
I think I can't accept indexRouter from app.ts but this is too much. That way, this file will contain only this single line and this is unacceptable
Solution: add proper routes here. Delete APIV1Router